### PR TITLE
dialogs: unify the dialog titles

### PR DESCRIPTION
**Pull request by jelly**
Unify them to all use the form `$action <b>$name</b>` so all dialog titles use the same style.

---

#### Editor

image

#### Delete

image

#### Rename

image

##### Permissions

image

### CODE REVIEW

File: src/dialogs/delete.tsx [hunk 1]
```diff
@@ -40,23 +41,25 @@ const ConfirmDeletionDialog = ({ dialogResult, path, selected, setSelected } : {
 
     let modalTitle;
     if (selected.length > 1) {
-        modalTitle = cockpit.format(forceDelete ? _("Force delete $0 items") : _("Delete $0 items?"), selected.length);
+        const selectedJSX = <b>{selected.length}</b>;
+        modalTitle = fmt_to_fragments(forceDelete ? _("Force delete $0 items") : _("Delete $0 items?"), selectedJSX);
```
Comment (cockpituous) [inline src/dialogs/delete.tsx:45]: This added line is not executed by any test.

File: src/dialogs/delete.tsx [hunk 2]
```diff
@@ -40,23 +41,25 @@ const ConfirmDeletionDialog = ({ dialogResult, path, selected, setSelected } : {
...
     } else {
         const selectedItem = selected[0];
+        const selectedJSX = <b>selectedItem.name</b>;
         if (selectedItem.type === "reg") {
-            modalTitle = cockpit.format(
-                forceDelete ? _("Force delete file $0?") : _("Delete file $0?"), selectedItem.name
+            modalTitle = fmt_to_fragments(
+                forceDelete ? _("Force delete file $0?") : _("Delete file $0?"), selectedJSX
```
Comment (cockpituous) [inline src/dialogs/delete.tsx:51]: This added line is not executed by any test.

File: src/dialogs/delete.tsx [hunk 3]
```diff
@@ -40,23 +41,25 @@ const ConfirmDeletionDialog = ({ dialogResult, path, selected, setSelected } : {
...
             );
         } else if (selectedItem.type === "lnk") {
-            modalTitle = cockpit.format(
-                forceDelete ? _("Force delete link $0?") : _("Delete link $0?"), selectedItem.name
+            modalTitle = fmt_to_fragments(
+                forceDelete ? _("Force delete link $0?") : _("Delete link $0?"), selectedJSX
```
Comment (cockpituous) [inline src/dialogs/delete.tsx:55]: These 2 added lines are not executed by any test.

File: src/dialogs/delete.tsx [hunk 4]
```diff
@@ -40,23 +41,25 @@ const ConfirmDeletionDialog = ({ dialogResult, path, selected, setSelected } : {
...
             );
         } else if (selectedItem.type === "dir") {
-            modalTitle = cockpit.format(
-                forceDelete ? _("Force delete directory $0?") : _("Delete directory $0?"), selectedItem.name
+            modalTitle = fmt_to_fragments(
+                forceDelete ? _("Force delete directory $0?") : _("Delete directory $0?"), selectedJSX
```
Comment (cockpituous) [inline src/dialogs/delete.tsx:59]: This added line is not executed by any test.

File: src/dialogs/delete.tsx [hunk 5]
```diff
@@ -40,23 +41,25 @@ const ConfirmDeletionDialog = ({ dialogResult, path, selected, setSelected } : {
...
             );
         } else {
-            modalTitle = cockpit.format(forceDelete ? _("Force delete $0") : _("Delete $0?"), selectedItem.name);
+            modalTitle = fmt_to_fragments(forceDelete ? _("Force delete $0") : _("Delete $0?"), selectedJSX);
```
Comment (cockpituous) [inline src/dialogs/delete.tsx:62]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 1 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:45]: This added line is not executed by any test.

File: src/dialogs/delete.tsx [hunk 6]
```diff
@@ -40,23 +41,25 @@ const ConfirmDeletionDialog = ({ dialogResult, path, selected, setSelected } : {
...
-            modalTitle = cockpit.format(
-                forceDelete ? _("Force delete file $0?") : _("Delete file $0?"), selectedItem.name
+            modalTitle = fmt_to_fragments(
+                forceDelete ? _("Force delete file $0?") : _("Delete file $0?"), selectedJSX
             );
         } else if (selectedItem.type === "lnk") {
-            modalTitle = cockpit.format(
-                forceDelete ? _("Force delete link $0?") : _("Delete link $0?"), selectedItem.name
+            modalTitle = fmt_to_fragments(
+                forceDelete ? _("Force delete link $0?") : _("Delete link $0?"), selectedJSX
```
Comment (cockpituous) [inline src/dialogs/delete.tsx:55]: This added line is not executed by any test.

File: src/dialogs/delete.tsx [hunk 7]
```diff
@@ -40,23 +41,25 @@ const ConfirmDeletionDialog = ({ dialogResult, path, selected, setSelected } : {
...
             );
         } else if (selectedItem.type === "dir") {
-            modalTitle = cockpit.format(
-                forceDelete ? _("Force delete directory $0?") : _("Delete directory $0?"), selectedItem.name
+            modalTitle = fmt_to_fragments(
+                forceDelete ? _("Force delete directory $0?") : _("Delete directory $0?"), selectedJSX
```
Comment (cockpituous) [inline src/dialogs/delete.tsx:59]: This added line is not executed by any test.

File: src/dialogs/delete.tsx [hunk 8]
```diff
@@ -40,23 +41,25 @@ const ConfirmDeletionDialog = ({ dialogResult, path, selected, setSelected } : {
...
             );
         } else {
-            modalTitle = cockpit.format(forceDelete ? _("Force delete $0") : _("Delete $0?"), selectedItem.name);
+            modalTitle = fmt_to_fragments(forceDelete ? _("Force delete $0") : _("Delete $0?"), selectedJSX);
```
Comment (cockpituous) [inline src/dialogs/delete.tsx:62]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 1 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:45]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 6 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:55]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 7 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:59]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 8 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:62]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 1 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:45]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 6 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:55]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 7 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:59]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 8 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:62]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 1 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:45]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 6 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:55]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 7 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:59]: This added line is not executed by any test.

File: src/dialogs/delete.tsx (hunk 8 above)
Comment (cockpituous) [inline src/dialogs/delete.tsx:62]: This added line is not executed by any test.

**Review by garrett — APPROVED** [review]
Thanks!

**Pull request merged**